public/app/core/components/PageNew/Page.tsx

```diff
@@ -167,6 +171,8 @@ const getStyles = (theme: GrafanaTheme2) => {
       boxShadow: shadow,
       background: theme.colors.background.primary,
       margin: theme.spacing(2, 2, 2, 1),
+      display: 'flex',
+      flexDirection: 'column',
```

Comment:
this is adding these back to pageInner (but this does not affect Page children as pageContent is now the wrapping class for children and it is a display block object with flexGrow: 1)